### ISSUE TITLE
AIY Voice Kit V1 - Is it possible start in local commands only?

### ISSUE THREAD

**Issue by grayerbeard** [google/aiyprojects-raspbian#686]

**Comment by grayerbeard** [comment]
My V1 been in cupboard a while but now got working although getting it to work complicated by issues with finding right versions of software and instructions.  (Will be posting in [my repository](https://github.com/grayerbeard/aiy) once fully sorted).
My issue now is that its hard to get "Local commands" to be reliable when using (say) "OK Google House Temperatures" to speak temperature readings fed in by MQTT from other R Pis.
Often its mis-heard and instead I get a long spoken answer to the mis interpreted question, in this case often the local weather.

So

Could I start in Local Commands only mode and only switch in to the full "Assistant mode" if I ask for it?   Anyone figured out how to do that ?

**Comment by manoj7410** [comment]
@grayerbeard Are you talking about the "local command" demo given at : https://github.com/google/aiyprojects-raspbian/blob/aiyprojects/src/examples/voice/assistant_library_with_local_commands_demo.py#L51 ?

**Comment by grayerbeard** [comment]
Yes could be would have to deviise a way to switch modes using the two different demos as a basis.

I have had a go with the "assistant_library_with_local_commands_demo.py" but every time you say one of your special commands you have a high risk that Google will misinterpret it as a request for info on something.  So for example I have other Pis monitoring controlling temperatures in the house.  If I say "OK Google House Temperatures" instead of getting the info I want soften it just tells me the weather report.    So it would be nice to be able to switch mode out of general assistant into a "My commands only mode".

So what I would love to manage is to switch between a mode where it only reacts to my limited list of commands and a mode where it will behave like a normal Google Assistant. i.e. at start you might say "OK Google Special Only" then it would only react to a limited list of my own commands until I say "OK Google Back to normal" when it would revert telling me tomorrows weather etc.

I am posting my attempts at [grayerbeard/aiy]( https://github.com/grayerbeard/aiy)
